app/lib/coursesHelper.coffee

```diff
@@ -1,6 +1,21 @@
 Levels = require 'collections/Levels'
 utils = require 'core/utils'
 
+courseShortName = (course) ->
```

Comment:
Refactored this out from method below.

Reply:
We actually have this refactored out in the [Course model](https://github.com/codecombat/codecombat/tree/master/app/models/Course.coffee#L17-L25), too. Should probably get it down to just one.

Reply:
Thank you for pointing this out to me.